modules/openapi/src/alloy/openapi/ExternalDocumentationMapper.scala

```diff
@@ -0,0 +1,112 @@
+/* Copyright 2022 Disney Streaming
+ *
+ * Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package alloy.openapi
+
+import _root_.software.amazon.smithy.jsonschema.JsonSchemaConfig
+import _root_.software.amazon.smithy.jsonschema.JsonSchemaMapper
+import _root_.software.amazon.smithy.jsonschema.Schema.Builder
+import _root_.software.amazon.smithy.model.shapes.Shape
+
+import scala.jdk.CollectionConverters._
+import software.amazon.smithy.model.traits.ExternalDocumentationTrait
+import software.amazon.smithy.model.node.ObjectNode
+import software.amazon.smithy.openapi.fromsmithy.OpenApiMapper
+import software.amazon.smithy.openapi.fromsmithy.Context
+import software.amazon.smithy.openapi.model.OpenApi
+import software.amazon.smithy.model.shapes.OperationShape
+import software.amazon.smithy.openapi.model.OperationObject
+import software.amazon.smithy.model.traits.Trait
+import software.amazon.smithy.openapi.model.ExternalDocumentation
+
+class ExternalDocumentationMapperOpenApi() extends OpenApiMapper {
+
+  override def after(context: Context[_ <: Trait], openapi: OpenApi): OpenApi =
+    if (context.getService.hasTrait(classOf[ExternalDocumentationTrait])) {
+      context.getService
+        .expectTrait(classOf[ExternalDocumentationTrait])
+        .getUrls()
+        .asScala
```

Comment:
I think this gives you a `Buffer` right? because you're calling getUrls which would return a list, if I understand the docs of External Documentation correctly.

That would mean: we get a list of links, we should insert an array of `Name -> Url` in the extensions object, line 51.

From the docs:
```smithy
@externalDocumentation(
    "Homepage": "https://www.example.com/"
    "API Reference": "https://www.example.com/api-ref"
)
service MyService {
    version: "2006-03-01"
}
```

Reply:
If I'm right, there are two other instances to be fixed in this file

Reply:
Ahhh, you're only picking the first one because the Open API specification has [one externalDocs extension objects](https://swagger.io/specification/#external-documentation-object)